client/my-sites/plans/jetpack-plans/jetpack-complete-page/components/cta-buttons/style.scss

```diff
@@ -0,0 +1,40 @@
+.jetpack-complete-page__cta-buttons {
+	display: inline-flex;
+	align-items: center;
+	justify-content: center;
+	width: 100%;
+	& > * {
+		flex: 1;
+	}
+	@include breakpoint-deprecated( "<960px" ) {
+		flex-direction: column;
+	}
+
+	.jetpack-complete-page__get-complete-button {
+		@include breakpoint-deprecated( ">960px" ) {
+			margin-right: 2em;
+		}
+		&.form-button {
```

Comment:
We need a rule inside this `.form-button` scope to specify that the Complete button has a `var(--studio-black)` border color, not the default green.